### ISSUE TITLE
Supported browser

### ISSUE THREAD

**Issue by naomioh** [MicrosoftDocs/azure-docs.ja-jp#2923]
I can see icons indicating the browsers are supported, but there is no description of the browser name.  I think it violates accessibility guidelines because screen reader users cannot get information about which browsers are supported.

---
#### ドキュメントの詳細

⚠ *このセクションを編集しないでください。 docs.microsoft.com で必須です ➟ GitHub の問題のリンク。*

* ID: deaf88eb-b88f-7110-2982-6f7bca367b6a
* Version Independent ID: b8309842-2c58-1d1c-f6e1-3dee50504191
* Content: [JavaScript Web アプリのための Azure Application Insights](https://docs.microsoft.com/ja-jp/azure/azure-monitor/app/javascript#feedback)
* Content Source: [articles/azure-monitor/app/javascript.md](https://github.com/MicrosoftDocs/azure-docs.ja-jp/blob/master/articles/azure-monitor/app/javascript.md)
* Service: **azure-monitor**
* Sub-service: **application-insights**
* GitHub Login: @mrbullwinkle
* Microsoft Alias: **mbullwin**

**Comment by srvbpigh** [comment]
Hello, @naomioh
 
Thank you for your feedback.
 
We are actively reviewing your comments and will get back to you soon.


Kind regards,
Microsoft DOCS International Team

**Comment by mrbullwinkle** [comment]
@naomioh the alt text for each of those icons contains the name of the associated browser so screen readers should be able to parse that without difficulty, but I will explicitly add the browser names to the body of the doc as well as it can't hurt to clarify if it was causing confusion.

**Comment by CeciAc** [comment]
Hi team, 
moving this to https://github.com/MicrosoftDocs/azure-docs/issues/43905 since the issue also affects the en-us version
